lib/board.js

```diff
@@ -1,138 +1,58 @@
 'use strict';
 
-var com = require('serialport');
-var when = require('when');
-var nodefn = require('when/node');
-
+var SerialPort = require('serialport').SerialPort;
 var bs2 = require('bs2-programmer');
+var bluebird = require('bluebird');
 
-function Board(){
```

Comment:
I agree currently theres no reason to have this be an object, but next week you'll probably want board.open and board.on('read') and board.write